en/phased-ranking.html

```diff
@@ -181,6 +186,70 @@ <h2 id="global-phase">Using a global-phase expression</h2>
 }
 </pre>
 
+<h2 id="normalizers">Cross-hit normalization in global-phase</h2>
+<p>
+  The ranking expressions configured for global-phase may perform
+  cross-hit normalization of input rank features or functions.  This
+  is designed to make it easy to combine unrelated scoring methods
+  into one final relevance score.
+  The syntax looks like a special pseudo-function call:
+  <ul>
+    <li> <code>normalize_linear(<em>my_function_or_feature</em>)</code> </li>
+    <li> <code>reciprocal_rank(<em>my_function_or_feature</em>)</code> </li>
+    <li> <code>reciprocal_rank(<em>my_function_or_feature</em>, <em>k</em>)</code> </li>
+    <li> <code>reciprocal_rank_fusion(<em>score_1</em>, <em>score_2</em> ...)</code> </li>
+  </ul>
+  The normalization will be performed across the hits that global-phase
+  reranks (see <a href="#globalphase-rerank-count">configuration</a> above).
+  This means that first, the input (<em>my_function_or_feature</em>)
+  is computed or extracted from each hit that global-phase will
+  rerank; then the normalization step is applied; afterwards when
+  computing the actual global-phase expression the normalized output
+  is used.
+  As an example, assume some text fields with bm25 enabled, an onnx
+  model like above, and a "popularity" numeric attribute:
+</p>
+<pre>
+    rank-profile with-normalizers inherits with-global-model {
```

Comment:
Were does the `with-global-model` come from? Is this something that we have built in now to signal global phase?